### PR TITLE
fix(explorer): experiment result view button conditions + bad file view style

### PR DESCRIPTION
**Pull request by davidlougheed**

### CODE REVIEW

File: src/components/explorer/IndividualExperiments.js [hunk 1]
```diff
@@ -52,24 +58,29 @@ const ExperimentResultActions = ({ result }) => {
...
-                    <FileDisplay uri={url} fileName={result.filename} />
-                )}
-            </Modal>
-            <Button size="small" icon="eye" onClick={onViewClick}>View</Button>{" "}
+                title={<span>View: {result.filename}</span>}
+                url={url}
+                fileName={result.filename}
+                hasTriggered={hasTriggeredViewModal}
+            />
+            <Tooltip title="View">
```
Comment (noctillion) [inline src/components/explorer/IndividualExperiments.js:81]: The button is not triggering the file download as expected. Could there be a conflict between the tooltip events and the onClick handler?

**Review by noctillion — CHANGES_REQUESTED** [review]

File: src/components/explorer/IndividualExperiments.js (hunk 1 above)
Comment (davidlougheed) [reply src/components/explorer/IndividualExperiments.js:81]: might be the case, i'll check it out!

Comment (davidlougheed) [reply src/components/explorer/IndividualExperiments.js:64]: should be fixed now

**Review by noctillion — APPROVED** [review]
lgtm

**Pull request merged**